package/gluon-core/files/etc/config/gluon

```diff
@@ -0,0 +1 @@
+config system system
```

Comment:
why not using `/etc/config/gluon-core` which is already supplied by gluon-core?

Reply:
I think calling the section 'core' would make sense, as this config is handled by *gluon-core* (and generally store config for package `gluon-X` in section `gluon.X`).

The commit title is not accurate, the top level of the UCI structure is called "package", so this would add a package "gluon" and a section "gluon.system" (or "gluon.core") to UCI.

Reply:
I plan to pick up this commit first (with my suggested changes), so the rest of Gluon can start using it. This PR will need to be adjusted to the changed section name.